### PR TITLE
[Release Preparation] Update workflow

### PR DESCRIPTION
**Pull request by alexander-smolyakov**
**Description**:
This pull request update commands in the Release Preparation workflow to match the [Version Bump](https://github.com/dbt-labs/actions/blob/main/.github/workflows/version-bump.yml) workflow.

_Changelog_:
- Use more up-to-date commands from `version-bump.yml` workflow;
- Replace `::set-output` commands;
- Format code;
- Update Bash scripts.

### CODE REVIEW

File: .github/workflows/release-prep.yml [hunk 1]
```diff
@@ -186,24 +187,48 @@ jobs:
         with:
           ref: ${{needs.create-new-branch.outputs.BRANCH_NAME}}
```
Comment (emmyoop) [inline .github/workflows/release-prep.yml:188]:
```suggestion
          ref: ${{ needs.create-new-branch.outputs.BRANCH_NAME }}
```

**Review by emmyoop — APPROVED** [review]
Looks good.  Can we add the PYTHON_VERSION env variable to this workflow like you have in the others?

File: .github/workflows/release-prep.yml (hunk 1 above)
Comment (alexander-smolyakov) [reply .github/workflows/release-prep.yml:188]: > Looks good. Can we add the PYTHON_VERSION env variable to this workflow like you have in the others?

Sure, I will add the `PYTHON_VERSION` env variable.

Comment (alexander-smolyakov) [reply .github/workflows/release-prep.yml:196]: Resolved in 8efd280ed1ded88cb1a886a0d6a089dc84c0155e

**Pull request merged**